src/components/table-ajax/__spec__.js

```diff
@@ -415,7 +421,9 @@ describe('TableAjax', () => {
   });
 
   describe("tags on component", () => {
-    let wrapper = shallow(
+    // TODO - 1007 - shallow seems to fail when creating Table
+    // data-component doesnt exist as prop on mount
+    let wrapper = mount(
```

Comment:
maybe this will fix the shallow render https://github.com/Sage/carbon/pull/1640/files#r156735929